iso.bbx

```diff
@@ -203,22 +270,30 @@
 \DeclareFieldFormat{mainsubtitle}{#1}
 \DeclareFieldFormat{issuesubtitle}{#1}
 
+% Format other / alternative titles in square brackets
 \DeclareFieldFormat{titleaddon}{\mkbibbrackets{#1}}
 \DeclareFieldFormat{booktitleaddon}{\mkbibbrackets{#1}}
 \DeclareFieldFormat{maintitleaddon}{\mkbibbrackets{#1}}
 
+% Format additional name information in square brackets
 \DeclareFieldFormat{nameaddon}{\mkbibbrackets{#1}}
 
-%name alias
+% NAME ALIAS
 
-\DeclareNameAlias{default}{last-first}
-\DeclareNameAlias{supervisor}{first-last}
+% Default name list format is last (family) followed by given (first) name
+\DeclareNameAlias{default}{family-given}
+% For supervisor, use inverted (natural) order format:
```

Comment:
I'd say `given-family` is *not* inverted (the `biblatex` manual uses the term 'reversed' but would otherwise agree with me here: "Jones, Edward" is reversed). 'Natural' is also a term I'd use for `family-given`. So the combination "inverted (natural)" is confusing.